app/styles/partials/event-card.scss

```diff
@@ -46,12 +46,29 @@
     object-fit: cover;
   }
 
-  .main.content {
-    display: block;
-    height: 130px;
+  @media only screen and (max-width: 578px) {
+    .main.content {
+      display: block;
+      height: 150px;
```

Comment:
setting it to 150px will again cause overflow if event title is long.
Eg - 
![2021-01-05 (1)_LI](https://user-images.githubusercontent.com/61330148/103580440-37d17400-4f00-11eb-8495-808616c97247.jpg)

Also, for shorter event names where 130px was previously enough now 20px is added extra.
Eg - 
![2021-01-05 (2)](https://user-images.githubusercontent.com/61330148/103580484-491a8080-4f00-11eb-8d65-cf2e0f5f71fc.png)

So there are two options - 
 
1.  add (...) if event name is long so that it always takes a constant space and then provide a constant height eg 130px.
2. to show the entire name, the height cannot be fixed, so you can remove the height constraint. (better option)

Reply:
Thanks. Working on it.